mc2/controllers/base/views.py

```diff
@@ -88,6 +88,7 @@ def form_valid(self, form):
         form.controller_form.instance.save()
 
         form.env_formset.instance = form.controller_form.instance
+        form.labels_formset.instance = form.controller_form.instance
```

Comment:
@calston since the override for `form_class` happens on the `DockerControllerCreateView`, it looks like we need to do this there instead of here